src/react-components/avatar-preview.js

```diff
@@ -63,6 +63,26 @@ function fitBoxInFrustum(camera, box, center, margin = DEFAULT_MARGIN) {
   camera.lookAt(center);
 }
 
+function getThemeBackground() {
+  const currentTheme = APP.store.state.preferences.theme;
+  const themes = window.APP_CONFIG?.theme.themes;
+  const defaultColor = 0xeaeaea;
+  if (currentTheme === "hubs-default") {
+    return defaultColor;
+  }
```

Comment:
I'd suggest guarding against these values possibly being undefined or empty instead: 

```suggestion
  if (!currentTheme || !themes || themes.length === 0) {
    return defaultColor;
  }
```